### PR TITLE
new recipes (pt. 2)

### PR DESCRIPTION
**Pull request by thomasrobiglio**
Add recipes for:
*create a random hypergraph
*cleanup
*add two hypergraph
*filterby
*Plot a hypergraph showing one order only
*Plot with stats

### CODE REVIEW

File: tutorials/cookbook/Recipes Thomas.ipynb [hunk 1]
```diff
@@ -0,0 +1,311 @@
+{
```
Comment (maximelucas) [inline tutorials/cookbook/Recipes Thomas.ipynb:1]: <p>Maybe add one line description of what we're trying to achieve with that recipe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (maximelucas) [inline tutorials/cookbook/Recipes Thomas.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    print(H_enron_cleaned

or just don't defined the new variable, I thin cleanup() works "in place"
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (maximelucas) [inline tutorials/cookbook/Recipes Thomas.ipynb:1]: <p>Maybe add one line description of what we're trying to achieve with that recipe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (maximelucas) [inline tutorials/cookbook/Recipes Thomas.ipynb:1]: <p>I think this is exactly what<code> H1 &lt;&lt; H2</code> does (lshift).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (maximelucas) [inline tutorials/cookbook/Recipes Thomas.ipynb:1]: <p>Maybe add one line description of what we're trying to achieve with that recipe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (maximelucas) [inline tutorials/cookbook/Recipes Thomas.ipynb:1]: <p>Maybe add one line description of what we're trying to achieve with that recipe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (thomasrobiglio) [reply tutorials/cookbook/Recipes Thomas.ipynb:1]: <pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; H1 = xgi.Hypergraph([[1,2,3],[3,4]])
&gt;&gt;&gt; H2 = xgi.Hypergraph([[1,2],[3,4],[4,5,6]])
&gt;&gt;&gt; h = H1 &lt;&lt; H2
&gt;&gt;&gt; print(h.edges.members())
[{1, 2, 3}, {3, 4}, {1, 2}, {3, 4}, {4, 5, 6}]

</pre><p>You see that the edges that is present in both hypergraphs is present twice in the merged hypergraph</p>

**Comment by thomasrobiglio** [comment]
Closing this as incorporated in #457

**Pull request closed without merging**